src/org/javarosa/core/model/RangeQuestion.java

```diff
@@ -32,4 +34,20 @@ public void setRangeStep(BigDecimal rangeStep) {
     public BigDecimal getRangeStep() {
         return rangeStep;
     }
+
+    public String getRangeStartLabel() {
+        return rangeStartLabel;
```

Comment:
Maybe it should return `rangeStart` if rangeStartLabel is not specified? It would be easier to use in Collect then.